enzyme/Enzyme/Enzyme.cpp

```diff
@@ -1705,7 +1704,7 @@ class Enzyme final : public ModulePass {
                                    : cast<StructType>(aug->fn->getReturnType())
                                          ->getElementType(tapeIdx);
         unsigned idxs[] = {(unsigned)tapeIdx};
-        Value *tapeRes = (tapeIdx == -1)
```

Comment:
?

Reply:
I have no idea how that occurred. Good catch.